### DISCUSSION TITLE
Bring back inaccurate suitable treatments

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15101] [Ideas and suggestions]
.

**Comment by Regalis11** [comment]
Could you elaborate on this? Are you referring to some recent change to the medic AI or the treatment suggestions on the health interface?

**Reply by A2ur3** [reply]
treatment suggestions on the health interface
```xml
<lowmedicalskillwarning>Low medical skill! The suitable treatments list may be inaccurate.</lowmedicalskillwarning>
```

**Reply by NotWendy** [reply]
I feel like the medical system being harder to learn for beginners isn't something Barotrauma needs right now. Inaccurate treatment suggestions can be pretty confusing for someone who's just trying to get their bearings. I'd be fine with it as a long-time player, but I can imagine it being a frustrating experience for someone who's new to the game.
But on a related note, it'd be cool if the medical scanner allowed you to view the exact stats of medical items. How much of which affliction they heal or add. Because right now your main resource for that is the wiki.
Maybe with both of these ideas combined, it'd work out? Low medical skill gives you inaccurate suggestions, but the Health Scanner HUD counters that when equipped, and lets you learn what each med actually does.
